### PR TITLE
Hotfix/clean

### PR DESCRIPTION
**Pull request by Sophie-Hawley**
Cleanup of most standing Info level alerts as reported by pylint. Shouldn't behave differently or have any less/more functionality vs Kyria/EsiPy/master

### CODE REVIEW

**Comment by Sophie-Hawley** [comment]
test failed in esipy/client.py, looking through it to see what happened standby

File: esipy/cache.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- encoding: utf-8 -*-
+'''-*- encoding: utf-8 -*-'''
```
Comment (Kyria) [inline esipy/cache.py]: This is wrong: this line is not a "comment" but define how the file is encoded for python and should start with `#`
https://www.python.org/dev/peps/pep-0263/

File: esipy/security.py [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-import base64
-import logging
-import time
+from .events import AFTER_TOKEN_REFRESH
+from .exceptions import APIException
+
+try:
+    from six.moves.urllib.parse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
```
Comment (Kyria) [inline esipy/security.py]: This is useless: you __need__ six for esipy, so it __can't__ do this exception. (if it does, you are doing something wrong). Also, this is the same import, just written differently as six mimics the structures of moved libs between py2/py3

Comment (Sophie-Hawley) [reply esipy/security.py]: ill have to look over my system for that one, was saying it couldn't import six for some reason

**Comment by Sophie-Hawley** [comment]
apologies for the insanity there, that should be all the info/warnings I can clear for the moment without firing a spanner into the tests again.

**Comment by Kyria** [comment]
I won't merge this commit, mostly because i've taken most of your changes manually to do them again / changing some name or words to better fit what I meant. 
I also fixed "everything else" and added a rcfile for pylint if you want to use it (currently only 1 thing don't pass pylint, but can't fix it since it's a pylint bug)

```shell
>> pylint --rcfile=.pylintrc esipy
Using config file E:\Dev\EsiPy\.pylintrc
************* Module esipy.security
E: 13, 0: Unable to import 'six.moves.urllib.parse' (import-error)

------------------------------------------------------------------
Your code has been rated at 9.85/10 (previous run: 9.85/10, +0.00)
```

Thanks for the PR anyway, even if I don't merge it (and as it made my use pylint...)

**Pull request closed without merging**